crates/gitbutler-branch-actions/src/remote.rs

```diff
@@ -95,10 +115,10 @@ pub fn list_local_branches(ctx: &CommandContext) -> Result<Vec<RemoteBranch>> {
             && branch.name.branch() != Some("gitbutler/edit")
             && branch.name.branch() != Some("gitbutler/target")
         {
-            remote_branches.push(branch);
+            branches.push(branch.to_owned());
```

Comment:
Could we make sure that the `branches` array is filled with the owned types already so we don't need to clone them?